lib/libzrepl/data_conn.c

```diff
@@ -830,6 +831,9 @@ uzfs_zvol_rebuild_dw_replica(void *arg)
 	if (wquiesce) {
 		uzfs_zvol_store_kv_pair(zinfo->clone_zv, STALE, 1);
 
+		VERIFY0(dsl_dataset_set_quorum(zinfo->main_zv->zv_name,
```

Comment:
better to add few comments here for quorum and stale also..
Also, comments for the need for quiescing at the end of this 'if' block